mantle/kola/harness.go

```diff
@@ -748,6 +749,12 @@ func registerTestDir(dir, testprefix string, children []os.FileInfo) error {
 				return errors.Wrapf(err, "reading %s", c.Name())
 			}
 			ignition = string(v)
+		} else if isreg && c.Name() == "config.fcc" {
+			b, err := exec.Command("fcct", fpath).Output()
```

Comment:
Might be worthwhile to add a comment along the lines of `TODO: add FCCT support to platform/conf` (and file a subsequent RFE for adding the support).